ocs_ci/deployment/deployment.py

```diff
@@ -2790,9 +2790,10 @@ def validate_dpa(self):
         3. backupstoragelocation resource in "Available" phase
 
         """
-        # Check restic pods
+        # Check restic pods.
+        # Restic pods have been renamed to node-agent
         restic_list = get_pods_having_label(
-            "name=restic", constants.ACM_HUB_BACKUP_NAMESPACE
+            "name=node-agent", constants.ACM_HUB_BACKUP_NAMESPACE
```

Comment:
Hi @shylesh, this needs to be conditionally handled based on OADP versions present on hub.
Changes from OADP 1.1 to 1.2: https://github.com/mateusoliveira43/oadp-operator/blob/fix/remove-1.1-leftovers/docs/upgrade1-1.md#changes-from-oadp-11-to-12

Reply:
ah ok, sorry, I missed it
I will update it.